covsirphy/cleaning/jhu_complement.py

```diff
@@ -199,17 +203,54 @@ def _recovered_full(self, df):
         if df[self.R].max() > self.max_ignored:
             # Necessary if sum of recovered is more than 99%
             # of sum of recovered and infected when outbreaking
-            sel_1 = df[self.C] > self.max_ignored
+            sel_C1 = df[self.C] > self.max_ignored
+            sel_R1 = df[self.R] > self.max_ignored
             sel_2 = df[self.C].diff().diff().rolling(14).mean() > 0
-            s_df = df.loc[
-                sel_1 & sel_2 & (df[self.R] > 0.99 * (df[self.C] - df[self.F]))]
-            if s_df.empty:
+            sel_3 = df[self.R] < 0.01 * (df[self.C] - df[self.F]).rolling(14).mean()
+            s_df_1 = df.loc[
+                sel_C1 & sel_2 & (df[self.R] > 0.99 * (df[self.C] - df[self.F]))]
+            s_df_2 = df.loc[sel_R1 & sel_2 & sel_3]
+            if s_df_1.empty and s_df_2.empty:
```

Comment:
Please consider
- `cf_diff = (df[self.C] - df[self.F]).rolling(14).mean()`,
- `sel_3 = df[self.R] < 0.01 * cf_diff`
- `sel_4 = df[self.R] > 0.99 * cf_diff` and
- `s_df_1 = df.loc[sel_C1 & sel_2 & sel_4`

Reply:
Applied changes!